jkube-kit/common-maven/src/test/java/org/eclipse/jkube/kit/common/util/MavenUtilTest.java

```diff
@@ -102,31 +108,27 @@ public void testGetDependencies(@Mocked MavenProject mavenProject) {
     final org.apache.maven.model.Dependency dep2 = dep1.clone();
     dep2.setArtifactId("artifact2");
     dep2.setType("jar");
-    new Expectations() {
-      {
-        mavenProject.getDependencies();
-        result = Arrays.asList(dep1, dep2);
-      }
-    };
+    List<org.apache.maven.model.Dependency> dependenciesList = new ArrayList<>();
+    dependenciesList.add(dep1);
+    dependenciesList.add(dep2);
+    Mockito.when(mavenProject.getDependencies()).thenReturn(dependenciesList);
     // When
     final List<Dependency> dependencies = MavenUtil.getDependencies(mavenProject);
     // Then
     assertThat(dependencies).hasSize(2);
   }
 
   @Test
-  public void testGetTransitiveDependencies(@Mocked MavenProject mavenProject) {
+  public void testGetTransitiveDependencies() {
     // Given
     final Artifact artifact1 = new DefaultArtifact("org.eclipse.jkube", "foo-dependency", "1.33.7",
         "runtime", "jar", "", new DefaultArtifactHandler("jar"));
     final Artifact artifact2 = new DefaultArtifact("org.eclipse.jkube", "bar-dependency", "1.33.7",
         "runtime", "jar", "", new DefaultArtifactHandler("jar"));
-    new Expectations() {
-      {
-        mavenProject.getArtifacts();
-        result = new HashSet<>(Arrays.asList(artifact1, artifact2));
-      }
-    };
+    Set<Artifact> artifactSet= new HashSet<Artifact>();
+    artifactSet.add(artifact1);
+    artifactSet.add(artifact2);
+    Mockito.when(mavenProject.getArtifacts()).thenReturn(artifactSet);
```

Comment:
Could be simplified to maybe:
```suggestion
    when(mavenProject.getArtifacts()).thenReturn(new HashSet<>(Arrays.asList(artifact1, artifact2)));
```

Reply:
Simplified!